assets/optional/multus/06-daemonset.yaml

```diff
@@ -0,0 +1,114 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus
+  namespace: openshift-multus
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the Multus networking component on each node.
+spec:
+  selector:
+    matchLabels:
+      app: multus
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: multus
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      serviceAccountName: multus
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: linux
+      priorityClassName: "system-node-critical"
+      tolerations:
+      - operator: Exists
+      initContainers:
+      - name: install-container-network-plugins
+        image: container-networking-plugins
+        imagePullPolicy: IfNotPresent
+        command: [ "/bin/bash", "-ec", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh;
+        volumeMounts:
+        - name: cnibin
+          mountPath: /host/opt/cni/bin
+        - mountPath: /entrypoint
+          name: cni-binary-copy
+        env:
+        - name: DEFAULT_SOURCE_DIRECTORY
+          # TODO: Change when microshift-specific multus image is ready
+          value: "/usr/src/plugins/rhel9/bin/"
+        - name: BINARIES_TO_COPY
+          value: "{bridge,ipvlan,macvlan,static,dhcp,host-local}"
```

Comment:
Q: `dhcp` will be supported in MicroShift (if so, you need to create DHCP plugin server pod as https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/multus/003-dhcp-daemon.yaml)